confd/etc/calico/confd/templates/bird6_ipam.cfg.template

```diff
@@ -38,6 +38,19 @@ function calico_export_to_bgp_peers(bool internal_peer) {
   if ( net ~ {{$cidr}} ) then { accept; }
   {{- end}}
 {{- end}}
+{{- $is_rr_key := printf "/bgp/v1/host/%s/rr_cluster_id" (getenv "NODENAME")}}
+{{- $lb_ips := "/bgp/v1/global/svc_loadbalancer_ips"}}
+{{- if exists $is_rr_key}}{{$is_rr := getv $is_rr_key}}
```

Comment:
nit, but I think `is_rr` should be named `rr_cluster_id` since `is_rr` implies a boolean value, but actually the value here is the RR cluster ID (if set, an empty otherwise)

Reply:
Fair enough, will change